### ISSUE TITLE
Add support for mhurdle objects (mhurdle package)

### ISSUE THREAD

**Issue by leifeld** [leifeld/texreg#40]
A basic version was provided by Yves Croissant on 17 March 2017 by e-mail:

```R
extract.mhurdle <- function (model, include.nobs = TRUE, ...){
  s <- summary(model, ...)
  names <- rownames(s$coefficients)
  class(names) <- "character"
  co <- s$coefficients[, 1]
  se <- s$coefficients[, 2]
  pval <- s$coefficients[, 4]
  class(co) <- class(se) <- class(pval) <- "numeric"
  n <- nobs(model)
  lik <- logLik(model)
  gof <- numeric()
  gof.names <- character()
  gof.decimal <- logical()
  gof <- c(gof, n, lik)
  gof.names <- c(gof.names, "Num. obs.", "Log Likelihood")
  gof.decimal <- c(gof.decimal, FALSE, TRUE)
  tr <- createTexreg(coef.names = names, coef = co, se = se, pvalues = pval,
                     gof.names = gof.names, gof = gof, gof.decimal = gof.decimal)
  return(tr)
}

setMethod("extract", signature = className("mhurdle", "mhurdle"), definition = extract.mhurdle)
```
The code needs to be checked and added, together with entries in the help files, vignette, and NAMESPACE.

**Comment by helix123** [comment]
I believe, these objects rather stem from package `mhurdle` (https://cran.r-project.org/package=mhurdle)

**Comment by leifeld** [comment]
Thanks for the hint, I have updated the title!

**Comment by czucca** [comment]
closed by PR leifeld#91
